unified-ai-platform-samples/README.md

```diff
@@ -0,0 +1,6 @@
+# Google Cloud AI Platform (Unified) Products
```

Comment:
Nit: Can we simplify the title to just "AI Platform (Unified)"?

(Similar to how https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/training/README.md just says "AI Platform Training" at the top)

I guess you probably copied "Products" in the title from the top level README, but technically AI Platform (Unified) is one product.